docs/user-guide.md

```diff
@@ -0,0 +1,72 @@
+# User Guide
+
+## Features
+
+### How to verify a kani harness using the extension
```

Comment:
"How to verify a kani harness using the extension" isn't a feature, but "Harness Listing" is. Try to come up with simple names for these features so it's clearer to users what can and cannot be done.